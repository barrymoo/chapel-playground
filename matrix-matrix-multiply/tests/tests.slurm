#!/usr/bin/env bash
#SBATCH --time=12:00:00
#SBATCH --job-name=tests
#SBATCH --output=tests.out
#SBATCH --nodes=9
#SBATCH --ntasks-per-node=1
#SBATCH --cluster=mpi

source /ihome/sam/bmooreii/src/chapel/util/setchplenv.bash
source /ihome/sam/bmooreii/projects/chapel/chapel.sh

export HFI_NO_CPUAFFINITY=1

$SLURM_SUBMIT_DIR/getNonLocaleSlice \
    --numLocales=$SLURM_NNODES \
    --tasksPerLocale=$SLURM_NTASKS_PER_NODE \
    --dimension=10
