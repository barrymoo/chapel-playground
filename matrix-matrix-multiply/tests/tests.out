Setting CHPL_HOME to /ihome/sam/bmooreii/src/chapel
Setting CHPL_HOST_PLATFORM to linux64
Updating PATH to include /ihome/sam/bmooreii/src/chapel/bin/linux64
                     and /ihome/sam/bmooreii/src/chapel/util
Updating MANPATH to include /ihome/sam/bmooreii/src/chapel/man
0
0
2 2 2
2 2 2
2 2 2
2 2 2
1 1 1
1 1 1
1 1 1
1 1 1
3 3 3 3
3 3 3 3
3 3 3 3
6 6 6 6
6 6 6 6
6 6 6 6

1
1
0 0 0 0
0 0 0 0
0 0 0 0
0 0 0 0
2 2 2
2 2 2
2 2 2
2 2 2
4 4 4
4 4 4
4 4 4
7 7 7
7 7 7
7 7 7

2
2
1 1 1
1 1 1
1 1 1
1 1 1
0 0 0 0
0 0 0 0
0 0 0 0
0 0 0 0
5 5 5
5 5 5
5 5 5
8 8 8
8 8 8
8 8 8

3
3
5 5 5
5 5 5
5 5 5
4 4 4
4 4 4
4 4 4
6 6 6 6
6 6 6 6
6 6 6 6
0 0 0 0
0 0 0 0
0 0 0 0
0 0 0 0

4
4
3 3 3 3
3 3 3 3
3 3 3 3
5 5 5
5 5 5
5 5 5
7 7 7
7 7 7
7 7 7
1 1 1
1 1 1
1 1 1
1 1 1

5
5
4 4 4
4 4 4
4 4 4
3 3 3 3
3 3 3 3
3 3 3 3
8 8 8
8 8 8
8 8 8
2 2 2
2 2 2
2 2 2
2 2 2

6
6
8 8 8
8 8 8
8 8 8
7 7 7
7 7 7
7 7 7
0 0 0 0
0 0 0 0
0 0 0 0
0 0 0 0
3 3 3 3
3 3 3 3
3 3 3 3

7
7
6 6 6 6
6 6 6 6
6 6 6 6
8 8 8
8 8 8
8 8 8
1 1 1
1 1 1
1 1 1
1 1 1
4 4 4
4 4 4
4 4 4

8
8
7 7 7
7 7 7
7 7 7
6 6 6 6
6 6 6 6
6 6 6 6
2 2 2
2 2 2
2 2 2
2 2 2
5 5 5
5 5 5
5 5 5

